ts/Series/Treegraph/TreegraphSeries.ts

```diff
@@ -0,0 +1,935 @@
+             *
+             * @type {boolean}
+             * @since next
+             * @default false
+             * @product highcharts
+             *
+             * @sample highcharts/series-treegraph/reversed-nodes
+             *         Treegraph series with reversed nodes.
+             */
+            reversed: false,
+            /**
+             * @extends   plotOptions.series.marker
+             * @excluding enabled, enabledThreshold
+             */
+            marker: {
+                radius: 10,
+                lineWidth: 0,
+                symbol: 'circle',
+                fillOpacity: 1,
+                states: {}
+            },
+            link: {
+                /**
+                 * The color of the links between nodes.
+                 *
+                 * @type {Highcharts.ColorString}
+                 * @private
+                 */
+                color: Palette.neutralColor60,
+                /**
+                 * The line width of the links connecting nodes, in pixels.
+                 * @type {number}
+                 *
+                 * @private
+                 */
+                lineWidth: 1,
+                /**
+                 * Radius for the rounded corners of the links between nodes.
+                 * Works for `default` link type.
+                 *
+                 * @private
+                 */
+                radius: 10,
+                cursor: 'default',
+                /**
+                 * Type of the link shape.
+                 *
+                 * @sample   highcharts/series-treegraph/link-types
+                 *           Different link types
+                 *
+                 * @type {'default' | 'curved' | 'straight'}
+                 * @product highcharts
+                 *
+                 */
+                type: 'curved'
+                /**
+                 * Modifier of the shape of the curved link. Works best for
+                 * values between 0 and 1, where 0 is a straight line, and 1 is
+                 * a shape close to the default one.
+                 *
+                 * @default 0.5
+                 * @type {number}
+                 * @since next
+                 * @product highcharts
+                 * @apioption series.treegraph.link.offset
+                 *
+                 */
+            },
+            /**
+             * Options applied to the collapse button.
+             */
+            collapseButton: {
+                onlyOnHover: true,
+                /**
+                 * Whether the button should be visible.
+                 * @default true
+                 */
+                enabled: true,
+                /**
+                 * Offset of the button in the x direction.
+                 * @default 0
+                 */
+                x: 0,
+                /**
+                 * Padding inside the button.
+                 * @default 2
+                 */
+                padding: 2,
```

Comment:
This doesn't work... I tried to set more padding, but it looks like only width and height works.